shuffle/cli/src/account.rs

```diff
@@ -13,17 +16,74 @@ use diem_sdk::{
 use diem_types::{
     account_config, chain_id::ChainId, transaction::authenticator::AuthenticationKey,
 };
+use directories::BaseDirs;
 use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
-use std::path::PathBuf;
+use std::fs;
+use std::io;
+use std::os::unix::fs::PermissionsExt;
+use std::path::Path;
+use structopt::StructOpt;
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
-    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+// Creates new account from randomly generated private/public key pair if key doesn't exist.
+// Rotates key if user already has one and passes in the rotate subcommand
+pub fn handle() -> Result<()> {
+    let base = &BaseDirs::new().unwrap();
+    let node_config_path = base.home_dir().join(".shuffle");
```

Comment:
Based on some due diligence on that `directories` crate, I think we should follow the standard shown here: https://github.com/dirs-dev/directories-rs#example

Specifically that `~/.config/shuffle` should be our destination, matching `~/.config/truffle`.

<img width="470" alt="CleanShot 2021-10-04 at 22 17 18@2x" src="https://user-images.githubusercontent.com/635121/135949874-bd076ed5-ec9c-48d5-8ebc-d65e4eea4a09.png">

Reply:
hmm if you're referring to `config_dir()` that `directories` offers, it only outputs the ~/.config/ on Linux. On mac and windows it gives some other path: "/Users/avinash00/Library/Application Support/". I can still hard code the output to be `~/.config/shuffle` tho!

Reply:
So IIUC the standard is based on a user's XDG config which is preferable over a hardcoding of a path because as a user i may configure my `.config` directory to be somewhere else.